### PR TITLE
CB-10503: Documentation link points to 'latest' version even in latest buils

### PR DESCRIPTION
**Pull request by sarangan12**
Fixed the link to point to dev version of the docs in development

@dblotsky @riknoll @rakatyal @nikhilkh Can you please review and merge this PR?

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ title:          Apache Cordova
 title_suffix:   "Apache Cordova"
 url:            "https://cordova.apache.org" # the base hostname & protocol
 baseurl:        "" # the subpath of the site, e.g. /blog; NOTE: no trailing slash
+docsversion:       "latest"
```
Comment (riknoll) [inline _config.yml]: Whitespace is weird here. Otherwise LGTM

**Comment by sarangan12** [comment]
@riknoll Fixed the spacing

**Comment by dblotsky** [comment]
We have another variable, `latest_docs_version`, whose name sounds like this one. Please rename this variable to something more explicit (like `default_linked_docs_version`) and please add it above [this line][line] so that it's grouped with the other one.

[line]: https://github.com/apache/cordova-docs/blob/master/_config.yml#L22

**Comment by riknoll** [comment]
+1 to @dblotsky's comment

**Pull request closed without merging**